frontend/src/components/course-learners-list/CourseLearnersList.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from './_course-learners-list.scss';
+import classNames from 'classnames/bind';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
```

Comment:
I skimmed over the fontawesome license, and it might be ok, but this is something we're going to have to be careful about